libtelemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -96,6 +97,18 @@ public void updateSessionIdRotationInterval(int hour) {
     }
   }
 
+  public boolean isDebugLoggingEnabled() {
+    return debugLoggingEnabled;
+  }
+
+  public void setDebugLoggingEnabled(boolean debugLoggingEnabled) {
```

Comment:
get/set nomenclature is hijacked by the Java language and should be only used when defining POJOs/DTOs.